tools/metilene/metilene.xml

```diff
@@ -0,0 +1,158 @@
+<?xml version="1.0" encoding="utf-8"?>
+<tool id="metilene" name="metilene" version="0.2-4.4">
+    <description>
+        calling differentially methylated regions from
+        bisulfite sequencing data
+    </description>
+    
+    <requirements>
+        <requirement type="package" version="0.2-4" >metilene</requirement>
+        <requirement type="binary">Rscript</requirement>
+        <requirement type="package" version="5.18.1" >perl</requirement>
+        <requirement type="package" version="3.2.1" >R</requirement>
+        <requirement type="package" version="2.24" >bedtools</requirement>
+        <requirement type="package" version="8.22">gnu_coreutils</requirement>
+    </requirements>
+    
+    <stdio>
+        <exit_code range=":-1"/>
+        <exit_code range="1:" />
+    </stdio>
+
+    <version_command>echo "0.2-4"</version_command>
+
+    <command><![CDATA[
+    ## =============== preprocessing
+    ##   generate metilene_g1_g2.input from multiple bedgraph files
+    metilene_input.pl 
+    --in1 $opt_in1
```

Comment:
`$opt_in1` is now space separated is this correct? Can this be handled if the user specifies n > 1 files?